salesagility/SuiteCRM
# Replace all uses of the deprecated db_utils functions.

## Description

The only remaining uses of these three functions is in their unit tests.

## Motivation and Context
The functions `db_convert`, `db_concat`, and `from_db_convert` are deprecated and have been for a while. They can simply be replaced by the counterpart functions in DBManager, but hadn't been as of yet.

https://github.com/salesagility/SuiteCRM/blob/fa1dbf3fb22666f5784e8c38286ac03b376bf66c/include/utils/db_utils.php#L44-L67

## How To Test This
Make sure the tests pass, probably do some basic manual testing to make sure things aren't broken.

## Types of changes
Technical debt payment I guess? :P


### Final checklist
